src/index.html

```diff
@@ -1,12 +1,50 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__body">
+    <div class="page__calendar">
+      <article
+        class="calendar
+          calendar--month-length--31
+          calendar--start-day--wed"
+      >
+        <a href="#" class="calendar__day"></a>
```

Comment:
would be better to do it with `ul` & `li`
```suggestion
      <ul
        class="calendar
          calendar--month-length--31
          calendar--start-day--wed"
      >
        <li class="calendar__day"></a>
```